sdk/test/NetStandard/IntegrationTests/IntegrationTests/TimestreamQuery.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Amazon.TimestreamQuery;
+using Amazon.TimestreamQuery.Model;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Amazon.DNXCore.IntegrationTests
+{
+    public class TimestreamQuery : TestBase<AmazonTimestreamQueryClient>
+    {
+        [Fact]
+        [Trait(CategoryAttribute, "TimestreamQuery")]
+        public async Task SkipsEndpointDiscoveryWithServiceUrl()
```

Comment:
This test doesn't create a VPC endpoint, but without the change in `EndpointDiscoveryResolver` it'd throw the same error as the original GitHub issue (`Requests will not succeed until an endpoint can be retrieved or an endpoint is manually specified`).